discovery_engine_core/Cargo.toml

```diff
@@ -1,19 +1,18 @@
 [workspace]
 members = [
     "ai/*",
+    "async-bindgen/*",
     "bindings",
     "core",
     "providers",
     "tooling",
 ]
 
 [patch.crates-io.async-bindgen]
-git = "https://github.com/xaynetwork/xayn_async_bindgen.git"
-branch = "main"
+path = "./async-bindgen/async-bindgen"
 
 [patch.crates-io.async-bindgen-gen-dart]
-git = "https://github.com/xaynetwork/xayn_async_bindgen.git"
-branch = "main"
+path = "./async-bindgen/async-bindgen-gen-dart"
```

Comment:
we should just change the dependencies in the package to same workspace path dependencies, the reason this are patch dependencies is because we originally considered publishing the async-bindgen tools